Makefile

```diff
@@ -275,7 +276,7 @@ protobuf: check-protoc
 
 .PHONY: react-init
 react-init:
-	cd frontend/ ; yarn install --frozen-lockfile
+	cd frontend/ ; yarn install --frozen-lockfile; yarn buildLib
```

Comment:
I don't think we should be building anything in `make react-init` - this command is just for setting up the frontend environment.

Reply:
Ah ok so just remove it since build commands shouldn't exist?

Reply:
Right, `react-init` should not build any of our code.